source/Test.cpp

```diff
@@ -97,6 +97,28 @@ namespace{
 		// SDL_PushEvent in the future to use the regular SDL event-handling loops.
 		return EventToUI(menuOrGamePanels, event);
 	}
+	
+	string ShipToString(const Ship &ship)
+	{
+		string shipDescription = "";
+		const System *shipSystem = ship.GetSystem();
+		const Planet *shipPlanet = ship.GetPlanet();
```

Comment:
I mean, this function does only one thing, so it's clear that `system`, `planet` and `description` refers to the ship's properties. So I'd drop the `ship` prefix imo.